packages/open-next/src/core/routingHandler.ts

```diff
@@ -144,9 +145,12 @@ export default async function routingHandler(
   // Api routes are not present in the routes manifest except if they're not behind /api
   // /api even if it's a page route doesn't get generated in the manifest
   // Ideally we would need to properly check api routes here
+  const apiPrefix = RoutesManifest.basePath
+    ? path.join("/", RoutesManifest.basePath, "api")
```

Comment:
Do we really need `path/posix` here ? A template literal is probably enough here unless i'm missing something
This part could run in a different runtime than node and `path/posix` might not be available there.

The `apiPrefix` could also be computed outside the function, no need to recompute this for every route. Not a big deal though

Reply:
> Do we really need `path/posix` here ? A template literal is probably enough here unless i'm missing something
> This part could run in a different runtime than node and `path/posix` might not be available there.

I was trying to be clever about supporting `basePath` values with different combinations of leading/trailing `/`. Given the [NextJS docs](https://nextjs.org/docs/app/api-reference/next-config-js/basePath) uses `/docs` as the example and that existing code already makes similar assumptions about `basePath`, a template literal works.